kuma/core/tests/test_validators.py

```diff
@@ -11,31 +13,31 @@ def test_valid_javascript_identifier(self):
         The function ``valid_javascript_identifier`` validates a given identifier
         according to the latest draft of the ECMAScript 5 Specification
         """
-        self.assertTrue(valid_javascript_identifier('hello'))
+        self.assertTrue(valid_javascript_identifier(b'hello'))
```

Comment:
It's so tempting to re-write this in the parametrized purest style. But I think this straightforward migration is best for getting Py3 done 👍